compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -226,12 +226,34 @@ void addGPUMatmulTensorCorePassPipeline(OpPassManager &pm,
 }
 
 void addGPUTransposePassPipeline(OpPassManager &pm) {
-  tileAndBufferize(pm);
```

Comment:
@ThomasRaoux Can you advise if this is the correct pipeline arrangement?  I've compared IR before/after the change. The mechanics of shared mem transposed are unchanged, however the IRs do differ in the use of memeref.subviews.

Reply:
yes this looks good